tools/testing-server/routes/mock-apis.js

```diff
@@ -110,7 +110,9 @@ module.exports = fp(async function (fastify, testServer) {
   fastify.get('/js', (request, reply) => {
     reply.type('text/javascript').send('console.log(\'hi\')')
   })
-  fastify.get('/text', (request, reply) => {
+  fastify.get('/text', {
+    compress: false
+  }, (request, reply) => {
```

Comment:
Should we just add this to all the mock api routes?

Reply:
Probably tbh